src/katsdpnetboxutilities/system/generate-report.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu
+NAME=$1
+mkdir -p docs/docs /reports/${NAME}
+./device-md.py --device-info http://sdp-services.sdp.kat.ac.za/servers -o ./docs/docs ${NAME}
```

Comment:
Could you write directly to /reports/${NAME}? Thus not needing the next line and part of line above. 
Just asking.

Reply:
it's actually possible to write to `/reports`. I removed the directory `/docs/docs`